targets/ar71xx-generic

```diff
@@ -3,9 +3,10 @@ config 'CONFIG_TARGET_SQUASHFS_BLOCK_SIZE=64'
 
 local ATH10K_PACKAGES = {}
 local ATH10K_PACKAGES_QCA9887 = {}
-if env.GLUON_WLAN_MESH == 'ibss' then
-	ATH10K_PACKAGES = {'-kmod-ath10k', 'kmod-ath10k-ct', '-ath10k-firmware-qca988x', 'ath10k-firmware-qca988x-ct'}
-	ATH10K_PACKAGES_QCA9887 = {'-kmod-ath10k', 'kmod-ath10k-ct', '-ath10k-firmware-qca9887', 'ath10k-firmware-qca9887-ct'}
+if env.GLUON_WLAN_MESH == '11s' then
+	ATH10K_PACKAGES = {'kmod-ath10k', '-kmod-ath10k-ct', 'ath10k-firmware-qca988x', '-ath10k-firmware-qca988x-ct'}
+	ATH10K_PACKAGES_QCA9886 = {'kmod-ath10k', '-kmod-ath10k-ct', 'ath10k-firmware-qca9888', '-ath10k-firmware-qca9888-ct'}
```

Comment:
is it on pupose to have qca9888 firmware in a variable called _QCA9886 ?

Reply:
Yes, as QCA9888 and QCA9886 share the same firmware and i haven't seen (yet) a device using a QCA9888.

However, i think we should add definitions for every chip (and include "QCA9880" to the "ATH10k_PACKAGES" variable name)

Reply:
ok, but with the big "K" to keep it consistent

Reply:
after a rebase, this could be resolved